docs/deploying/deploy-smart-contracts.mdx

```diff
@@ -62,7 +62,7 @@ eg: `yarn verify --network sepolia`
 
 This uses [etherscan-verify from hardhat-deploy](https://www.npmjs.com/package/hardhat-deploy#4-hardhat-etherscan-verify) to verify all the deployed contracts.
 
-You can alternatively use [hardhat-verify](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-verify) to verify your contracts by running `yarn hardhat-verify --network network_name contract_address`
+You can alternatively use [hardhat-verify](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-verify) to verify your contracts by running `yarn hardhat-verify --network network_name contract_address "Constructor arg 1"`
```

Comment:
Maybe this is overkill, because they probably should just stick to `etherscan-verify`, but we could add an explanation for new developers to know constructor args are optional:

Something like:

You can alternatively use [hardhat-verify](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-verify) to verify your contracts, passing network name, contract address and constructor arguments (if any): `yarn hardhat-verify --network network_name contract_address "Constructor arg 1"`

Reply:
Yup makes sense, tysm Pablo update it at https://github.com/scaffold-eth/se2-docs/pull/54/commits/97561c074b7b8731433a8528a6401e63fc1768da 🙌